src/tree/node.rs

```diff
@@ -34,6 +34,9 @@ pub struct Node
     /// `item` is the inner value which is holded by a Node.
     item: Goban,
     depth: usize,
+    last_move: BitBoard,
+    player_captures: u8,
```

Comment:
?

Reply:
This is to keep track of the number of capture already performed. It's needed here (could be in goban instead maybe?) because we have to know how many captures one of the two players has already done for every state in the game.